Exiled.API/Features/Player.cs

```diff
@@ -490,6 +490,15 @@ public Vector3 Position
             set => ReferenceHub.TryOverridePosition(value, Vector3.zero);
         }
 
+        /// <summary>
+        /// Gets or sets the relative player's position.
```

Comment:
Add a remark that the value will be 0,0,0,0 if they are not an FPC role.

Reply:
Okay

Reply:
done